brainscore_language/__init__.py

```diff
@@ -40,7 +42,8 @@ def load_benchmark(identifier: str) -> Benchmark:
     # imports to load plugins until plugin system is implemented
     from brainscore_language.plugins.wikitext_next_word_prediction import benchmark
     from brainscore_language.plugins.futrell2018 import benchmark
-    from brainscore_language.plugins.schrimpf2021 import benchmark
+#    from brainscore_language.plugins.schrimpf2021 import benchmark
```

Comment:
```suggestion
```
remove dead code
(sorry, this line was added erroneously by me to setup for the summer school tutorial)

Reply:
Committed suggestion.